tests/RobotFramework/tests/tedge/apt_plugin.robot

```diff
@@ -0,0 +1,69 @@
+*** Settings ***
+Documentation       Two settings have been specifically added to tedge-apt-plugin.
+...    tedge-apt-plugin list --name $PATTERN lists the packages which name matches the regular expression $PATTERN.
+...    tedge-apt-plugin list --maintainer $PATTERN lists the packages which maintainer matches the regular expression $PATTERN.
+...    If both --name and --maintainer patterns are provided, tedge-apt-plugin lists packages which either the name or the maintainer matches.
+...    These two settings can stored in /etc/tedge/tedge_config.toml in an apt table.
+...    If stored in /etc/tedge/tedge_config.toml, then the apt.name and apt.maintainer settings are used to filter packages listed to the cloud.
+...    No support is provided by tedge config so store apt.name and apt.maintainer.
+
+Resource            ../../resources/common.resource
+Library             ThinEdgeIO
+
+Suite Setup         Setup
+Suite Teardown      Get Logs
+
+
+*** Test Cases ***
+
+
+tedge-apt-plugin list
+    ${apt_list}    Execute Command    /etc/tedge/sm-plugins/apt list
+    Should Contain    ${apt_list}    apt
+    Should Contain    ${apt_list}    wget
+    Should Contain    ${apt_list}    dpkg
```

Comment:
Are these packages installed in the setup or just assuming that they exist?

Reply:
I took them out of the log of the test, so yes they exist